types/smart-truncate/index.d.ts

```diff
@@ -0,0 +1,6 @@
+// Type definitions for smart-truncate 1.0
+// Project: https://github.com/millerized/smart-truncate
+// Definitions by: Omer Yalhi <https://github.com/oyalhi>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export function smartTruncate(text: string, length: number, options?: { position?: number; mark?: string }): string;
```

Comment:
There is a bug here: smartTruncate is a default export, see here: https://github.com/millerized/smart-truncate/blob/master/src/smart-truncate.js#L51

So, this should be "export default function smartTruncate".